common/source/docs/common-underwater-sonars-landingpage.rst

```diff
@@ -11,6 +11,7 @@ Follow the links below (or in the sidebar) for configuration information based u
 .. toctree::
     :maxdepth: 1
 
+    Kogger Sonar <common-kogger-sonar>
```

Comment:
Please place in list in alphabetical order, thanks...will final build test when you believe you are finished...thanks

Reply:
Thank you for your comment!